### ISSUE TITLE
[2.0.17 Bug] Cannot save on mobile & tablet URGENT

### ISSUE THREAD

**Issue by joomleb** [JoomShaper/helix-ultimate#377]
Hi @mi-prakash 
from Chrome for Android, tested on 6" & 10.4"
there is no way to save anymore the template.

To switch to "desktop mode" is not anymore a workaround, the save button in the header is still outside the screen.

Unfortunately the events forced us to work 99% of time from mobile.
Please, Can you help with a quick workaround?

**Comment by mi-prakash** [comment]
@joomleb 
Thanks for your feed back. 
Unfortunately there is no quick way around. The admin template settings part is not fully responsive for mobile view yet.
We recommend you to use it in pc or in tablets (ie. bigger screen).
Hopefully in future we will make the user experience better.
Thank you

**Comment by joomleb** [comment]
@mi-prakash 
Yes, there is a quick way to temporally fix it (waiting for a better optimized version). It is really simple:
to move the Save button on the right into the header. (And, maybe, vice-versa the HU icon on the right)

Just it, it would be sufficient as a start point...

Do you agree ?

**Comment by joomleb** [comment]
@mi-prakash 
Please Can you help to have the "Save" button visible in the header in any way?  
It's creating a lot of problems for us, till 2.0.16 it was working, not good, but working...

**Comment by mi-prakash** [comment]
@joomleb 
we will check on this issue on the next update but can not guarantee that it will be fixed yet

**Comment by joomleb** [comment]
@mi-prakash 
Thank you for your answer. To just have the. save button visible into the Header would be sufficient for now.

It seems like the Header is not responsive and it has a fixed minimum width, so the Save button on the right is going "outside" the screen...

Please, What is the file we can take a look for the Header layout ?

**Comment by mi-prakash** [comment]
@joomleb 
this is a quick fix for now.
got to `ROOT/pluginss/system/helixultimate/assets/css/admin/helix-ultimate.css` and open the file

Copy & paste this code at the end of the file
`@media (max-width:576px){.hu-topbar .topbar-left,.hu-topbar .topbar-middle,.hu-topbar .topbar-right{min-width:auto}}@media (max-width:768px){.hu-topbar .topbar-left,.hu-topbar .topbar-middle,.hu-topbar .topbar-right{min-width:auto}}`

just like this show in the image
<img width="1184" alt="image" src="https://github.com/JoomShaper/helix-ultimate/assets/42798816/2ce1f23c-901a-4d51-80af-7f38bb2791eb">

This should fix the button issue
let me know if this works

**Comment by joomleb** [comment]
@mi-prakash 
in first of all many thanks.
Then, remember to consider that all my tests are done with mobile Chrome for Android (11) on smartphone 6.3" and tablet 10.4.

Now, following your suggestion, I can confirm that the right solution for the "Save" button in all conditions is to does not care about @media and just to add:

.hu-topbar .topbar-left,.hu-topbar .topbar-middle,.hu-topbar .topbar-right{min-width:auto}

Practically, to change the min-width definitions in the file with "auto'.

To have an usable HU on mobile still there are 2 awful issues:

**2** - When I turn the smartphone/tablet to the horizontal view,  the HU (the page) Is fixed and it is not vertically scrollable anymore. So, the HU menu is cut off:
https://photos.app.goo.gl/pYzt6tP7wWVgFiBa7

**3** - When I open the HU menu on smartphone vertical view the menu is opened on the left.
Sure it is because there is not sufficient space to open it on the right, anyway would be better to force it to open on the right, so I can view it as opened and turn to horizontal view to use HU:
https://photos.app.goo.gl/E2MhMYK3ui1KS1rU8

**Comment by mi-prakash** [comment]
@joomleb 
In the next update you will get the responsive theme settings. We are working on it.
so have patience and wait for the next update. Hopefully all your problems regarding this will be fixed.
Thank you

**Comment by joomleb** [comment]
Hi @mi-prakash 
with Helix Ultimate 2.1.0 + Joomla 4.4.4
the mobile view has been introduced, thank you. But there are some bugs:

**1** - In mobile vertical view [the HU modal box lost the X (close) button](https://photos.app.goo.gl/mQMHQuKw5oLsaCfH6)

**2** - if turn phone horizontally the HU modal box is hidden

**3** - When I save, [the Success Alert is not horizontally centered](https://photos.app.goo.gl/nod3QKQTYkAbmoPL7)

Please, Can you confirm them ?

**Comment by joomleb** [comment]
Hi @mi-prakash 
Helix Ultimate 2.1.1 + Joomla 4.4.4
Bugs:are still there:

**1** - In mobile vertical view [the HU modal box lost the X (close) button](https://photos.app.goo.gl/mQMHQuKw5oLsaCfH6)

**2** - if I turn the phone horizontally the HU modal box is hidden and I cannot scroll down the page

**3** - When I save, [the Success Alert is not horizontally centered](https://photos.app.goo.gl/nod3QKQTYkAbmoPL7)

Please, Can you confirm them ?

**Comment by mi-prakash** [comment]
@joomleb 
yes, 
in the last version we focused on new features. on the future version we will provide bug fixes
Thank you

**Comment by joomleb** [comment]
@mi-prakash 
I understand. 
Anyway, meanwhile, Please, Can you confirm the 3 bugs as described  above ?

And, if confirmed, Please, Can you add the red "Bug" label to this ticket ?
